src/ServiceMonitor/ServiceMonitor.cpp

```diff
@@ -191,7 +193,14 @@ HRESULT Service_Monitor::StopServiceByName(LPCTSTR pServiceName, DWORD dwTimeOut
         }
         else
         {
-            _tprintf(L"\nERROR: Failed to stop or query status of service '%s' error [%x]\n", pServiceName, hr);
+            if (serviceStopped)
+            {

```

Comment:
Can you explain what is the value of the additional error message, while the "else" branch still has two possibilities.

Reply:
If service monitor fails to get the current status of the target service, it is clearly error and the error message should start with "ERROR:". However, if that happens after stopping the service, it is kind of expected. In that case, we can ignore it and that's why I changed the message to use "INFORMATION:" instead of "ERROR:"